common/utils/helpers.py

```
@@ -0,0 +1,38 @@
+from django.contrib.auth.models import User
+from django.test import TestCase
+from rest_framework.exceptions import PermissionDenied
+from rest_framework.test import APIRequestFactory
+
+from applications.API.models import Insult, InsultCategory
+
+
+def _check_ownership(self, obj, user):
```

Ruff (0.12.2)

Missing return type annotation for private function `_check_ownership`

Add return type annotation: `None`

(ANN202)